assignment.js

```diff
@@ -66,6 +80,29 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+let numbers = [57, 13, 4, 38]
+let currentLargestNumber = 0
+
+/* numbers.forEach(function(number){
+  // console.log(numbers) ** checking to see if it is within scope
+  let largestNumber = Math.max(57, 13, 4, 38)
+  console.log(`${largestNumber} is the highest number in this array.`)
+}) */
+
+//if (true) {
+  // console.log(numbers)
+  //let currentLargestNumber = numbers[3]
+  //console.log(currentLargestNumber)
+//}
+
+for (let number of numbers) {
+  console.log(number)
+  let largestNumber = Math.max(numbers[i])
+  console.log(largestNumber)
+}
+
+// help 
```

Comment:
🛠 Read the comment below and update your code accordingly

@nurisellie while there are multiple ways to do this, I suggest you take the following approach: 

Exploring using  `.sort()` on the array. Using sort an array of numbers will automatically sort them in "ascending" order:

```
let numbers = [5, 22, 4]
let sortedNumbers = numbers.sort() // <= [4, 5, 22]
```
Then you can either:
* use your knowledge of arrays to grab the last element in the sortedNumbers array,
* or you can "reverse" (hint hint) the  sortedNumbers array so the numbers will be ordered in "descending" order and then grab the 1st element in the reverse sorted array